presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -46,24 +46,23 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Symbol
         }
 
         WindowNode windowNode = (WindowNode) node;
-        Collection<FunctionCall> actualCalls = windowNode.getWindowFunctions().values();
+        List<FunctionCall> actualCalls = windowNode.getWindowFunctions().values().stream().map(function -> function.getFunctionCall()).collect(toList());
 
         if (actualCalls.size() != functionCalls.size()) {
             return false;
         }
 
         LinkedList<FunctionCall> expectedCalls = new LinkedList<>(functionCalls);
```

Comment:
Thanks to having this list of type `LinkedList` the remove a few lines below (https://github.com/prestodb/presto/pull/5785/files#diff-0af664c27fd891a04814795944ca138bR58) may be of `O(1)` complexity, instead of possible `O(n)` with other implementations of array (e.g. `ArrayList`).